api/types/events/events.proto

```diff
@@ -1996,6 +1996,7 @@ message OneOf {
         events.MySQLDebug MySQLDebug = 86;
         events.MySQLRefresh MySQLRefresh = 87;
         events.SFTP SFTP = 88;
+        events.SessionRecordingAccess SessionRecordingAccess = 89;
```

Comment:
The message number should match the one in master, it's fine to skip them here

Reply:
Ah okay, I wasn't sure